File: AnvilConvert.js
let AnvilconvertJar;
let PrefixAnvilcon = "[AnvilConvert] ";
function enable(){
    console.info(PrefixAnvilcon+"§eLoading...");
    AnvilconvertJar = new NnClassLoader({ urls: ["https://github.com/Awzaw/AnvilConverter/releases/download/v1.0/AnvilConverter.jar"] });
    manager.createCommand("regionconvert", "Convert MCregion worlds to Anvil!", AnvilCmd, "Use /regionconvert <world>", [], "anvil.convert.world");
    console.info(PrefixAnvilcon+"§aDone!");
}
function disable(){
    console.info(PrefixAnvilcon+"§cBye!");
};
function AnvilCmd(sender, args, label){
    if(!sender.hasPermission("anvil.convert.world")){
        sender.sendMessage("§c"+server.getLanguage().translateString("commands.generic.permission"));
        return;
    }
    if(args.length < 1){
        sender.sendMessage("§c"+server.getLanguage().translateString("commands.generic.usage", "/regionconvert <world>"));
        return;
    }
    try{
      let main = AnvilconvertJar.type("net.minecraft.world.level.storage.AnvilConverter");
      main.main([server.dataPath+"worlds", args[0]]);
    }catch(error){
        sender.sendMessage("§c"+server.getLanguage().translateString("commands.generic.exception"));
        console.info("§cError in command regionconvert", error);
    }
}
module.exports = {
    onEnable: enable,
    onDisable: disable
}

File: src/main/java/org/CreadoresProgram/AnvilConvert/AnvilConvert.java
package org.CreadoresProgram.AnvilConvert;
import net.minecraft.world.level.storage.AnvilConverter;
import cn.nukkit.plugin.PluginBase;
import cn.nukkit.command.Command;
import cn.nukkit.command.CommandSender;
public class AnvilConvert extends PluginBase {
    public String prefix = "[AnvilConvert] ";
    @Override
    public void onEnable() {
        this.getLogger().info(this.prefix + "§eLoading...");
        this.getServer().getCommandMap().register("regionconvert", new CommandConver());
        this.getLogger().info(this.prefix + "§aDone!");
    }
    @Override
    public void onDisable() {
        this.getLogger().info(this.prefix + "§cBye!");
    }
    public class CommandConver extends Command {
        public CommandConver(){
            super("regionconvert", "Convert MCregion worlds to Anvil!", "/regionconvert <worldname>", new String[]{});
            this.setPermission("anvil.convert.world");
        }
        @Override
        public boolean execute(CommandSender sender, String label, String[] args){
            if(!sender.hasPermission("anvil.convert.world")){
                sender.sendMessage("§c"+getServer().getLanguage().translateString("commands.generic.permission"));
                return true;
            }
            if(args.length < 1){
                sender.sendMessage("§c"+getServer().getLanguage().translateString("commands.generic.usage", "/regionconvert <world>"));
                return true;
            }
            String worldName = String.join(" ", args);
            AnvilConverter.main(new String[]{ getServer().getDataPath()+"worlds", worldName});
            return true;
        }
    }
}